definitions/ext-jfrog-servers/definition.yml

```diff
@@ -0,0 +1,21 @@
+# Derived from the JFROG log ingestion, denotes the server where 
+domain: EXT
+type: JFROG_SERVER
+
+synthesis:
+  identifier: hostname
```

Comment:
Would this hostname be unique within an account scope ?
If not -  maybe there is an unique attribute or you could compose one like [here](https://github.com/newrelic/entity-definitions/blob/ea5063824c8911029f3755718353110d1e988e16/definitions/ext-pixie_dns/definition.yml#L6) ?

Reply:
Yes, the hostname is unique here. It is the identifier for the server being used.